Update docs (#2982)
edgelesssys/constellation: README.md

### 🔒 Everything always encrypted

> * Runtime encryption: All nodes run inside AMD SEV-based Confidential VMs (CVMs). Support for Intel TDX is already available for Azure and more CSPs will be added in the future.

Reviewer: ```suggestion
* Runtime encryption: All nodes run inside Confidential VMs (CVMs) based on AMD SEV or Intel TDX.
```
Let's keep it short at this place.
